test/cri-containerd/policy_test.go

```diff
@@ -28,7 +28,7 @@ func securityPolicyFromContainers(policyType string, containers []securitypolicy
 	if err != nil {
 		return "", err
 	}
-	policyString, err := securitypolicy.MarshalPolicy(policyType, false, pc, []securitypolicy.ExternalProcessConfig{}, true)
+	policyString, err := securitypolicy.MarshalPolicy(policyType, false, pc, []securitypolicy.ExternalProcessConfig{}, true, true)
```

Comment:
parameters to `MarshalPolicy` keep expanding, we should consider taking a `PolicyConfig` or something.

Reply:
I brought up the same topic in conversation just the other day. Once everything is in, we should look at using a builder object of some kind, or a PolicyConfig as you say.